app/models/book.rb

```diff
@@ -1,4 +1,13 @@
 class Book < ApplicationRecord
   belongs_to :author
   belongs_to :publisher, polymorphic: true
+
+  validates :title, :author_id, :publisher_id, presence: true
+  validates :price, numericality: { greater_than_or_equal_to: 1, allow_nil: false }
```

Comment:
Just wanted to make sure am not selling any books for less than 1 dollar or else ill lose my business 👍